packages/core/src/components/graphs/meter.ts

```diff
@@ -0,0 +1,98 @@
+// Internal Imports
+import { Component } from "../component";
+import { DOMUtils } from "../../services";
+import { Tools } from "../../tools";
+
+// D3 Imports
+import { select } from "d3-selection";
+import { scaleLinear } from "d3-scale";
+
+
+export class Meter extends Component {
+	type = "meter";
+
+	render(animate = true) {
+		const self = this;
+		const svg = this.getContainerSVG();
+		const options = this.model.getOptions();
+		const displayData = this.model.getDisplayData();
+
+		// meter only deals with 1 dataset (like pie/donut)
+		const dataset = displayData.datasets[0];
```

Comment:
If we deal with 1 dataset only then we should create a separate model that extends `ChartModel` and remove the concept of `datasets`